drivers/gles3/storage/texture_storage.cpp

```diff
@@ -1389,8 +1389,24 @@ void TextureStorage::texture_debug_usage(List<RS::TextureInfo> *r_info) {
 		tinfo.format = t->format;
 		tinfo.width = t->alloc_width;
 		tinfo.height = t->alloc_height;
-		tinfo.depth = t->depth;
 		tinfo.bytes = t->total_data_size;
+
+		switch (t->type) {
+			case Texture::TYPE_3D:
+				tinfo.depth = t->depth;
+				tinfo.bytes *= t->depth;
```

Comment:
`total_data_size` is already multiplied by `depth` does this not make the reported bytes much too high?


See:
https://github.com/godotengine/godot/blob/61598c5c88d95b96811d386cb20d714c35f4c6d7/drivers/gles3/storage/texture_storage.cpp#L854

The same goes for layered textures

Reply:
Ah right, I was confused since layered textures' data size gets overriden with the size of a single layer at
https://github.com/godotengine/godot/blob/61598c5c88d95b96811d386cb20d714c35f4c6d7/drivers/gles3/storage/texture_storage.cpp#L1524

It looks like 3d textures aren't affected by this, however.

Reply:
Oof, I didn't notice that. The fix looks great though